polyglot/piranha/src/utilities/tree_sitter_utilities.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
 };
 use colored::Colorize;
 use itertools::Itertools;
-use log::info;
+use log::{info};
```

Comment:
Am I missing something about `log::{info}` vs `log::info`

Reply:
No its equivalent. 
I think during my changes I had added a warn macro in this file and made this import -> 
`log::info` ->  log::{info, warn}`. But then I removed `warn`, and `{}` remained. I will clean i up.